scrapers/scrape_ge.py

```diff
@@ -31,6 +31,25 @@
     print(dd_test)
     is_first = False
 
+
+# get hospitalized number
+hosp_url = 'https://www.hug.ch/coronavirus-maladie-covid-19/situation-aux-hug'
```

Comment:
But this is only one hospital, right? I'm not sure, but I guess there must be others as well.

Reply:
I wasn't really sure in the first place. But it seems that it's not a single hospital from the plural name :)
So it's also linked from the [GE covid webpage](https://www.ge.ch/covid-19-situation-epidemiologique-geneve-informations-du-canton/situation-epidemiologique-geneve).
And the values in the [graph](https://www.hug.ch/coronavirus-maladie-covid-19/situation-aux-hug) from 30th of October seem to match with what is [in the repository](https://github.com/openZH/covid_19/blob/master/fallzahlen_kanton_total_csv_v2/COVID19_Fallzahlen_Kanton_GE_total.csv#L282) (458 hospitalizations)